docs/library/processors/delays/allpassdelay/example-1.py

```diff
@@ -0,0 +1,26 @@
+from signalflow import *
+graph = AudioGraph()
+
+#-------------------------------------------------------------------------------
+# Using AllpassDelay to add a dreamy atmosphere to synth arpeggios
+#-------------------------------------------------------------------------------
+clock = Impulse(3.5)
+
+Am7 = [ 67, 64, 60, 57 ] * 4
+D7 = [ 62, 66, 69, 72] * 4
+arpeggios = Am7 + D7
+sequence = Sequence(arpeggios, clock)
+frequency = MidiNoteToFrequency(sequence)
+
+oscillator = SquareOscillator(frequency)
+envelope= ASREnvelope(0.1, 0, 0.2, 1.0, clock)
+voice = oscillator * envelope * 0.3
+filtered = SVFilter(voice, SIGNALFLOW_FILTER_TYPE_LOW_PASS, 4000, 0.3)
```

Comment:
Although you can specify the filter type using the constant names (`SIGNALFLOW_FILTER_TYPE_LOW_PASS` etc), there are string equivalents which let you do the same thing, and I think are more readable:

`low_pass`, `high_pass`, `notch`, etc

So let's switch to using `low_pass` here and in other examples.

Reply:
When I try using `low_pass`, I get the following error:
`NameError: name 'low_pass' is not defined`
Any idea why the string equivalents aren't being recognised?

Reply:
"low_pass" as a string  - `SIGNALFLOW_FILTER_TYPE_LOW_PASS` is the name of the constant, "low_pass" is the value it contains

Reply:
Ah gotcha! All works